cypress/support/commands.ts

```diff
@@ -19,6 +19,31 @@ Cypress.Commands.add('login', (username: string, password: string) => {
   });
 });
 
+Cypress.Commands.add(
+  'signup',
+  (username: string, email: string, password: string) => {
+    cy.visit('http://localhost:3000/sign-up');
+    cy.get('form').within(() => {
+      const randomStr = (Math.random() + 1).toString(36).substring(7);
+      cy.get('#username').type(`KevinLu${randomStr}`);
```

Comment:
Should this be using `username` instead?

Reply:
I changed #username to id=username like you did in login test